rocAL/rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -439,40 +546,55 @@ extern "C"  RocalTensor  ROCAL_API_CALL rocalLensCorrection(RocalContext context
 /// \param is_output
 /// \return
 extern "C"  RocalTensor  ROCAL_API_CALL rocalLensCorrectionFixed(RocalContext context, RocalTensor input,
-                                                                float strength, float zoom, bool is_output);
+                                                                 float strength, float zoom, bool is_output,
+                                                                 RocalTensorLayout rocal_tensor_output_layout = ROCAL_NHWC,
+                                                                 RocalTensorOutputType rocal_tensor_output_type = ROCAL_UINT8);
 
 ///
 /// \param context
 /// \param input
 /// \param is_output
 /// \return
-extern "C"  RocalTensor  ROCAL_API_CALL rocalPixelate(RocalContext context, RocalTensor input, bool is_output );
+extern "C" RocalTensor ROCAL_API_CALL rocalPixelate(RocalContext context, RocalTensor input,
+                                                    bool is_output,
+                                                    RocalTensorLayout rocal_tensor_output_layout = ROCAL_NHWC,
+                                                    RocalTensorOutputType rocal_tensor_output_datatype = ROCAL_UINT8);
+
 
 ///
 /// \param context
 /// \param input
 /// \param is_output
 /// \param shift
 /// \return
-extern "C"  RocalTensor  ROCAL_API_CALL rocalExposure(RocalContext context, RocalTensor input, bool is_output,
-                                                        RocalFloatParam shift = NULL);
+extern "C" RocalTensor ROCAL_API_CALL rocalExposure(RocalContext context, RocalTensor input,
+                                                    bool is_output,
+                                                    RocalFloatParam shift = NULL,
+                                                    RocalTensorLayout rocal_tensor_output_layout = ROCAL_NHWC,
+                                                    RocalTensorOutputType rocal_tensor_output_datatype = ROCAL_UINT8);
 
 /// \param context
 /// \param input
 /// \param is_output
 /// \param shift
 /// \return
-extern "C"  RocalTensor  ROCAL_API_CALL rocalExposureFixed(RocalContext context, RocalTensor input, float shift, bool is_output);
+extern "C" RocalTensor ROCAL_API_CALL rocalExposureFixed(RocalContext context, RocalTensor input,
+                                                         float shift, bool is_output,
```

Comment:
Please rename the param names as per the appropriate function call
I think we have named most of the params as shift/alpha before. Please check and change appropriately if applicable